frontend/src/routes/SingleShowing/BuyModal.js

```diff
@@ -46,9 +45,15 @@ const BuyModal = ({
     );
   }
 
-  const { participantInfo, bioklubbnummer, sfBuyLink } = buyData;
+  const { participantInfo, bioklubbnummer, sfBuyLink, showing } = buyData;
   const { ticketsBought } = showing;
 
+  const renderForetagsbiljetter = () => {
+    return (<div><SmallHeader>Företagsbiljetter</SmallHeader>
+      {showing.participants.filter(p => p.payment && p.payment !== 'swish').map(p => <CopyValue key={p.payment} text={p.payment} />)}
```

Comment:
Builds on @Edholm's comment, but maybe we should handle `företagsbiljetter` differently. It feels weird with it sometimes being a `string` and sometimes an `array`.